README.rst

```diff
@@ -0,0 +1,55 @@
+====================
+dspace-python-client
+====================
+
+A Python client library for working with the DSpace API. This was written against the `DSpace version 6.x API <https://wiki.lyrasis.org/display/DSDOC6x/REST+API>`_ and will not work with later versions of DSpace.
+
+-----
+Usage
+-----
+
+This project has not been released for active use yet. Check back soon!
+
+------------
+Development
+------------
+
+^^^^^^^^^^^^
+Dependencies
+^^^^^^^^^^^^
+~~~~~~
+Python
+~~~~~~
+
+Version 3.9
+
+~~~~~~
+Poetry
+~~~~~~
+This project uses Poetry for dependency management and packaging.
+
+You only need to install Poetry once, so if you've already installed it for another project you can skip this step. Poetry will automatically pick up the current Python version and use it to create virtualenvs accordingly.
+
+To install poetry:
+
+* Mac/unix::
```

Comment:
FYI I used `brew install poetry`. I'm not sure if the default curl install handles updates but I know brew will so I felt better about that path.

Reply:
The poetry docs explicitly recommend against that. I wonder if that's part of what caused the python version issues? 

```
Using alternative installation methods will make Poetry always use the Python version for which it has been installed to create virtualenvs.

So, you will need to install Poetry for each Python version you want to use and switch between them.```

Reply:
Yeah that's definitely not how it seems to be working for me. I didn't need to reinstall poetry, I just needed to switch to the version of python I wanted and then it worked. I think with the dot file for the python version I wouldn't have run into this. Documenting the official path is fine... still curious if it handles updates but either way 👍🏻